.github/workflows/operator.yml

```diff
@@ -227,7 +248,7 @@ jobs:
 
   e2e:
     name: End-to-end
-    needs: [build-operator, build-rclone, build-restic, build-rsync, build-syncthing, kubectl-plugin]
+    needs: [build-operator, build-rclone, build-restic, build-rsync, build-syncthing, build-scorecard, kubectl-plugin]
```

Comment:
`build-scorecard` should be moved down to `e2e-success.needs` since it's not actually used during the e2e tests.